benchmarks/dynamo/runner.py

```diff
@@ -264,6 +265,27 @@ def parse_args():
         default=DASHBOARD_DEFAULTS["dashboard_gh_cli_path"],
         help="Github CLI path",
     )
+    parser.add_argument(
+        "--batch_size", type=int, default=None, help="batch size for benchmarking"
+    )
+    parser.add_argument(
+        "--threads",
+        "-t",
+        type=int,
+        default=None,
+        help="number of threads to use for eager",
```

Comment:
This only controls the eager mode?

Reply:
No. It controls the eager and inductor. The help info is updated.